src/Result/inc/Result.h

```diff
@@ -15,10 +16,16 @@ namespace calculator
     class Result
     {
         public:
-            Expression expression;
-            float answer;
-            std::string fullResult;
-            void returnResult();
+            Result(Expression expression, float answer, std::string fullResult) : expression_(expression), answer_(answer), fullResult_(fullResult){}
+
+            /**
+             * @returns the full result string
+            */
+            std::string getFullResult();
+        private:
+            Expression expression_;
```

Comment:
These should be in alphabetical order and need to be aligned
```cpp
float       answer_;
Expression  expression_;
std::string fullResult_;
```
Make sure all private members in all your classes are like this

Reply:
Note on my personal preference: I don't really like aligning variables like this. But it is the Dune way and sometimes you do things you don't like because of project coding styles. 🤷